src/main/java/ml/codeboy/thebot/commands/sound/Join.java

```diff
@@ -0,0 +1,18 @@
+package ml.codeboy.thebot.commands.sound;
+
+import ml.codeboy.thebot.events.CommandEvent;
+import ml.codeboy.thebot.util.Util;
+
+public class Join extends AudioCommand {
+    public Join() {
+        super("join");
+    }
+
+    @Override
+    public void run(CommandEvent event) {
+        if(ensureConnected(event)) {
+            event.reply(Util.sign(event.getBuilder().setTitle("Joined voicechannel successfully"), event));
+            return;
+        }
```

Comment:
You can't just ignore errorhandling. If the bot can't join you should at least let the user know about it